.travis.yml

```diff
@@ -23,3 +23,10 @@ install:
 
 script:
   - ./gradlew build testDownstream -PcheckJava6Compatibility --daemon -s -i && ./gradlew ciPerformRelease
+
+after_failure:
+  - ./after-failure.sh
+
+#TODO Added here only for tests purposes because failure is harder to reproduce. If it's still here - remove it!!
```

Comment:
Shouldn't it be removed before merge (looking at the comment :) )?

Reply:
Yes, I will remove it as soon as it works correctly on the pull request branch